apps/silverback-gatsby/gatsby-config.mjs

```diff
@@ -36,7 +37,7 @@ export const plugins = [
       graphql_path: process.env.DRUPAL_GRAPHQL_PATH,
       auth_key: process.env.DRUPAL_AUTH_KEY,
       schema_configuration: './graphqlrc.yml',
-      directives: { echo, responsiveImage },
+      directives: { echo, responsiveImage, responsiveImageSharp },
```

Comment:
I think it should be enough to just pass `responsiveImage: responsiveImageSharp` here? Then there are no changes to the schema necessary.

Reply:
actually, I kept it like that because in silverback-gatsby we used both, as an example. We use sharp for articles and cloudinary for guternberg pages. But we could change the naming as well.